notebooks/001-getting-started/001-getting-started.ipynb

```diff
@@ -54,10 +54,13 @@
     "\n",
```

Comment:
<p>What do you mean with index one?</p><p> </p><p>a quick look <s>of</s> at its config file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>